pkg/controller/vsphere/reconciler.go

```diff
@@ -560,6 +576,18 @@ func clone(s *machineScope) (string, error) {
 		return "", handleVSphereError(multipleFoundMsg, notFoundMsg, defaultError, err)
 	}
 
+	hwVersion, err := getHwVersion(s.Context, vmTemplate)
+	if err != nil {
+		// TODO throw an event also?
+		klog.Warningf("Unable to detect machine template HW version for machine %s: %v", s.machine.GetName(), err)
+	}
+	if hwVersion == 13 { // TODO figure out this condition more precisely
```

Comment:
While not likely, the hardware version could be less than 13 if the template hw version is downgraded.  Also, might consider replacing the raw hardware version with a constant.

Reply:
I found this: https://github.com/openshift/openshift-docs/pull/42232/files
Seems it's reasonable to prevent machines creation with hw version lower than 15 for ocp 4.11+.